hicommonwealth/commonwealth
# user can now delete image in topic separately

## Link to Issue
Closes: #8841 

## Description of Changes
- User is now able to delete an image from a topic within the `edit_topic_modal` by clicking the "Remove image" button instead of having to manually delete the markdown. 
**NOTE: This solution solves mostly for 1 image. It works for 2, but the user first has to delete the first image if they also want to delete the second image. Please let me know if you'd like for me to change the code to account for multiple images.** 

## "How We Fixed It"

-added logic to look for `![image](<url>)` in the markdown and to display an image if present. And when "Remove image" is clicked the topic gets updated without the image markdown text. 

## Test Plan
- make yourself an admin
- go to Topics in sidebar
- create a topic and add an image. You can also add text if you want
- to make your life easier, make sure you check "Featured topic and sidebar"
- after creating the topic go to Topics > Manage Topics and click the pencil icon of the topic you just created
- confirm that you now see the image in the Edit topic modal
- click "Remove image" and confirm that the image disappears and that the markdown is gone 
- save the changes and confirm you see the toast that reads "Topic updated"

## Deployment Plan

1. 

## Other Considerations

-
